templates/addons/s3/policy.yml

```diff
@@ -0,0 +1,23 @@
+{{.BucketResourceName}}AccessPolicy:
+  Type: AWS::IAM::ManagedPolicy
+  Properties:
+    Description: !Sub
+      - Grants CRUD access to the S3 bucket ${Bucket}
+      - { Bucket: !Ref {{.BucketResourceName}} }
+    PolicyDocument:
+      Version: 2012-10-17
+      Statement:
+        - Sid: {{.BucketResourceName}}S3ObjectActions
```

Comment:
Let's just call this S3ObjectActions, that way we don't have to worry about sanitizing the bucket name.